src/components/Dasboard/AddButton/AddButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import Fab from "@material-ui/core/Fab";
+import AddIcon from "@material-ui/icons/Add";
+import Slide from "@material-ui/core/Slide";
+import { Grid, ClickAwayListener } from "@material-ui/core";
+import SlidingButton from "./SlidingButton";
+
+interface CreateButton {
```

Comment:
We can put interfaces in the respective model directory and let's try to make more generics interfaces. 
Also please follow the naming convention that the project is using. Interfaces should begin with I `ICreateButton`